CHANGELOG.md

```diff
@@ -26,10 +28,35 @@ New Features
 
     return function (datetime, logLevel, component, message, data) {
       method(datetime, logLevel, component, message, data);
-      // Send to your own server
+
+      // check for signaling events that previously were
+      // emitted on (now deprecated) eventListener
+      if (message === 'event' && data.group === 'signaling') {
+        if (data.name === 'waiting') {
+          assert.equal(data.level, 'warning');
```

Comment:
Should we remove these assertions and provide an example data object instead?

Reply:
I have now removed the asserts against `data.level` - But I am not sure if we should reproduce the documentation of the events object here. They are documented in https://media.twiliocdn.com/sdk/js/video/releases/2.8.0/docs/global.html
I tried to keep the sample similar to the original one that shows how to use `eventListener`